Apps/WebClient/src/NewClientApp/src/App.vue

```diff
@@ -20,11 +25,49 @@ const isHeaderVisible = computed(
         appStore.appError === undefined &&
         !currentPathMatches(loginCallbackPath, vaccineCardPath)
 );
+
+const isMobile = computed<boolean>(() => appStore.isMobile);
+
+function initializeResizeListener(): void {
+    window.addEventListener("resize", onResize);
+    onResize();
+}
+
+function onResize(): void {
+    windowWidth.value = window.innerWidth;
+
+    if (windowWidth.value < ScreenWidth.Mobile) {
+        if (!isMobile.value) {
+            setIsMobile(true);
+        }
+    } else {
+        if (isMobile.value) {
+            setIsMobile(false);
+        }
+    }
+}
+
+function setIsMobile(isMobile: boolean): void {
+    appStore.setIsMobile(isMobile);
+}
+
+onMounted(async () => {
+    windowWidth.value = window.innerWidth;
+
+    await nextTick();
+    initializeResizeListener();
+    initialized.value = true;
+});
+
+onBeforeUnmount(() => {
+    window.removeEventListener("resize", onResize);
+});
 </script>
 
 <template>
     <v-app>
         <HeaderComponent v-if="isHeaderVisible" class="d-print-none" />
+        <SidebarComponent class="d-print-none" />
```

Comment:
On one of Declan's PR's, he moved the `d-print-none` class onto the root element inside the component.  Might be cleaner to do the same here.

Reply:
Changed.